src/FtlPlayer.ts

```diff
@@ -1,15 +1,44 @@
 import { Janus, PluginHandle, Message, JSEP } from "janus-gateway";
 
+type Options = {
+    debug?: boolean,
+    hideMediaControls?: boolean
+}
+
 export class FtlPlayer {
     private static readonly janusPluginPackage = "janus.plugin.ftl";
+    private readonly logPrefix = "janus-ftl-player:";
     private element: HTMLVideoElement;
     private serverUri: string;
     static isJanusInitialized: boolean = false;
     private janusInstance: (Janus | null) = null;
     private janusPluginHandle: (PluginHandle | null) = null;
+    private options: Options;
+
+    constructor(element: HTMLVideoElement, serverUri: (string | null) = null, options: Options) {
+        // Allow configuring additional options
+        this.options = {
+            debug: true,
+            hideMediaControls: true
+        };
+        Object.assign(this.options, options);
 
-    constructor(element: HTMLVideoElement, serverUri: (string | null) = null) {
         this.element = element;
+
+        if (this.options.hideMediaControls) {
+            let uniqueClass = "janus-ftl-player-" + Math.floor(Math.random() * 10000)
```

Comment:
Instead of injecting a new stylesheet with a unique class containing properties that will only be set on a single element, consider just adding those properties to the element's style attribute via [`element.style.setProperty(name, value)`](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleDeclaration/setProperty).

Reply:
Honestly I couldn't figure out how to access advanced selectors like `::-webkit-media-controls-timeline`, if you can figure out how I'm happy to use that instead.

Reply:
Ah, yeah, that's a bit unclear to me too :D If we figure it out later we can change it. Thanks!